app/index.js

```diff
@@ -13,15 +13,16 @@ var workspace = require('loopback-workspace');
 var Workspace = workspace.models.Workspace;
 var bluemix = require('../bluemix/helpers');
 var fs = require('fs');
-var actions = require('../lib/actions');
+// var actions = require('../lib/actions');
```

Comment:
**In the first commit**, please DO NOT review the change in file `app/index.js`.

I have to update it to test the `acl` generator from cli, while the app generator is not finished.